src/styles/_placeholder.scss

```diff
@@ -0,0 +1,19 @@
+%arrow-position {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+}
+
+%animation-minutes-arrow {
+  animation: watch 60s linear infinite;
+}
+
+%animation-hour-arrow {
+  animation: watch 3600s linear infinite;
```

Comment:
move animation duration to the variables

Reply:
updated